### PR TITLE
Proper notification messages added while resetting password

### PR DESCRIPTION
**Pull request by akhil-kn**
### Description of the Change

Currently no notifications displayed while reset the password but some codes present for notification purposes. Now fixed the issues properly as possible.

screenshot1
 
### Benefits

screenshot2

screenshot3

screenshot4

### Possible Drawbacks

NA

### CODE REVIEW

File: core/jazz_ui/src/app/pages/login/oss/login.component.ts [hunk 1]
```diff
@@ -256,17 +256,11 @@ export class LoginComponent implements OnInit {
...
-                        
                     }, err => {
                         let error = JSON.parse(err._body);
-                        let errorMessage=this.toastmessage.errorMessage(err, 'reset');
-                        try{
-                            errorMessage = error.message;
-                        } catch(e) {
-                            console.log(e);
-                        }
+                        let errorMessage=this.toastmessage.errorMessage(err, 'resetPwd');
```
Comment (supritat) [inline core/jazz_ui/src/app/pages/login/oss/login.component.ts]: Consider a condition where toast message for unknown error is not defined in toast message. In such case use the message from API. (Same in other update password condition)

File: core/jazz_ui/src/config/toastmessages.ts [hunk 2]
```diff
@@ -95,7 +95,15 @@ export const toastMessage = {
         "error400":"Your registration failed. Seems like something went wrong.",
         "error401":"Token expired",
         "error404":"Your registration failed. Seems like something went wrong.",
-        "error500":"Your registration failed. Seems like something went wrong."},
+        "error500":"Your registration failed. Seems like something went wrong."
+    },
+    "resetPwd":{
+        "success":"A verification code is sent to your registered email.",
+        "error400":"Cannot reset password for the user as there is no registered/verified email or phone_number.",
```
Comment (supritat) [inline core/jazz_ui/src/config/toastmessages.ts]: Remove "phone_number". As of now the registration and verification happen only through email.

Comment (akhil-kn) [reply core/jazz_ui/src/config/toastmessages.ts]: Updated.

File: core/jazz_ui/src/app/pages/login/oss/login.component.ts (hunk 1 above)
Comment (akhil-kn) [reply core/jazz_ui/src/app/pages/login/oss/login.component.ts]: Handled by using try catch.

**Pull request merged**